types/react-navigation/index.d.ts

```diff
@@ -262,7 +263,7 @@ export type NavigationStackScreenOptions = NavigationScreenOptions & {
   headerTitleStyle?: Style,
   headerTintColor?: string,
   headerLeft?: React.ReactElement<any>,
-  headerBackTitle?: string,
+  headerBackTitle?: string | null,
```

Comment:
@phanalpha Ah, sorry, I forgot `null` was a valid value to pass for `headerBackTitle` specifically. Thanks for correcting me, this looks good.